ap2-receiver.py

```diff
@@ -846,6 +846,16 @@ def upgrade_to_encrypted(self, client_address, shared_key):
                 exit(-1)
     except Exception:
         print("[!] Network interface not found")
+        print("Available network interfaces:")
```

Comment:
Agree - look at e.g. args.netiface, and if it's absent, call this code block there (also).
Call can also be in the exception block, altho from a readability perspective, it might not be immediately clear under which conditions the exception is triggered and reached.